assets/sass/02-tools/mixins.scss

```
@@ -77,3 +77,21 @@
 		color: var(--button--color-text-active);
 	}
 }
+
+@mixin innerblock-margin-clear() {
+
+	// Clear the top margin for the first-child.
+	> *:first-child {
+		margin-top: 0;
+	}
+
+	// Last child that is not the appender.
+	> *:last-child():not(.block-list-appender) {
```

```suggestion
	> *:last-child:not(.block-list-appender) {
```

`last-child` doesn't use parens, so this is an invalid selector.

---

Good catch, thanks!